src/components/calcite-tooltip-manager/calcite-tooltip-manager.tsx

```diff
@@ -152,7 +154,8 @@ export class CalciteTooltipManager {
   focusEvent = (event: FocusEvent, value: boolean): void => {
     const tooltip = this.queryTooltip(event.target as HTMLElement);
 
-    if (!tooltip) {
+    if (!tooltip || tooltip === this.clickedTooltip) {
+      this.clickedTooltip = null;
```

Comment:
Just curious if this can use `composedPath`, since it already gives you the elements the event traversed, instead of capturing and storing the reference?

Reply:
I tried that but it doesn't give me the original event. It only gives me the focus event and I need to know if it came from a click.

Reply:
I see. Never mind then. :)